Backpack/Icon/Classes/BPKIconView.h

```diff
@@ -47,10 +47,20 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * The size of the icon currently being displayed. Setting updates
  * the displayed icon.
+ *
  * @see BPKIconSize
  */
 @property(nonatomic, assign) BPKIconSize size;
 
+/**
+ * Sets the size of the icon currently being displayed. Setting updates
+ * the displayed icon.
+ *
+ * @deprecated Setting size during the component lifecycle will soon be unsupported.
+ * @see BPKIconSize
+ */
+-(void)setSize:(BPKIconSize)size __deprecated_msg("Setting size during the component lifecycle will soon be unsupported.");
+
```

Comment:
I've explicitly declared `setSize:` so that we can mark it as deprecated without deprecating the readable property